src/main/java/net/minestom/server/inventory/click/InventoryClickProcessor.java

```diff
@@ -241,9 +241,13 @@ public InventoryClickResult dragging(@Nullable Inventory inventory, @NotNull Pla
             if (button == 0) {
                 // Start left
                 this.leftDraggingMap.put(player, new IntOpenHashSet());
+
+                clickResult = startCondition(inventory, player, slot, ClickType.START_LEFT_DRAGGING, clicked, cursor);
```

Comment:
Shouldn't all those new conditions check for cancellation?

Reply:
I'm not quite sure. The original code had this line at the declaration of the variable and doesn't seem to do any checks regarding cancellation either. Only when looping over the slots is a check done, which is still there.

The `startCondition` method does seem to do some checks itself regarding cancellation (namely setting cancel and refresh to `true` on the click result), so perhaps that is enough?